app/controllers/repairs_controller.rb

```diff
@@ -0,0 +1,35 @@
+class RepairsController < ApplicationController
+    def index
```

Comment:
In Ruby, each indentation should be [indented exactly 2 spaces](https://github.com/rubocop-hq/ruby-style-guide#spaces-indentation), not 4 spaces and not a full tab. Please fix the indentation throughout. (If you're using an IDE like VS Code, you can update this in your settings so that it will do this automatically for all Ruby files.)